src/utils/parse-arguments-into-options.ts

```diff
@@ -1,8 +1,36 @@
 import type { Args, RawOptions } from "../types";
 import arg from "arg";
+import * as https from "node:https";
```

Comment:
Maybe just do  : 

```ts
import * as https from "https"
```

Otherwise I get this warning in when I run `yarn dev` : 

![Screenshot 2024-05-22 at 9 23 29 PM](https://github.com/scaffold-eth/create-eth/assets/80153681/488005e0-b41f-4aee-a546-35a77ade8138)

or maybe add "node:https" in external array in rollup.config.js as suggested [here](https://github.com/rollup/rollup/issues/4365#issuecomment-1020897616)

Reply:
Not sure why my IDE imported it like that haha. I don't think we need the prefix in this case. 90889261c975fe9a73c0c0440cbf82ed78ce4f2e